olive/systems/system_config.py

```diff
@@ -31,6 +31,10 @@ class AzureMLTargetUserConfig(TargetUserConfig):
     aml_compute: str
     aml_docker_config: AzureMLDockerConfig
     instance_count: int = 1
+    # read timeout in seconds for HTTP requests, user can increase if they find the default value too small.
+    # The default value from azureml sdk is 3000 which is too large and cause the evaluations and pass runs to
+    # sometimes hang for a long time between retries of job stream and download steps.
+    read_timeout: int = 60
```

Comment:
Please add the description into Olive docs if it is exposed to users.

Reply:
We don’t have explanation of any of the system config parameter. So there’s not where to add them right now.

i think when we use pydantic fields like discussed in the team meeting, we can make the comment here into a description. It will get automatically documented them.